docs/snowpark/snowpark-backend-joins.md

```diff
@@ -0,0 +1,178 @@
+     dataSetA
+        |> innerJoin dataSetB
+            ( \a b -> a.id == b.id )
+        |> List.map (\ (x, y) -> 
+            { idA = x.id
+            , idB = y.id
+            })
+```
+
+The transformation to Snowpark will be:
+
+```Scala
+    def simpleInnerJoin
+        (dataSetA: com.snowflake.snowpark.DataFrame)
+        (dataSetB: com.snowflake.snowpark.DataFrame)
+        (implicit sfSession: com.snowflake.snowpark.Session): com.snowflake.snowpark.DataFrame = {
+    val dataSetAColumns: deparments.Joins.TypeA = new deparments.Joins.TypeAWrapper(dataSetA)    
+    val dataSetBColumns: deparments.Joins.TypeB = new deparments.Joins.TypeBWrapper(dataSetB)
+    
+    dataSetA.join(
+      dataSetB,
+      (dataSetAColumns.id) === (dataSetBColumns.id),
+      "inner"
+    ).select(
+      dataSetAColumns.id.alias("idA"),
+      dataSetBColumns.id.alias("idB")
+    )
+  }
+```
+
+You can notice that `innerJoin` function from **Morphir.SDK** is transform to `join` function of **Snowpark** passing the argument `inner` to choose the join type.
+
+The projection is changed to a **Snowpark select** to return a dataframe with the selected columns.
+
+### Left Join
+
+The `leftJoin` function is similar to `innerJoin`. You can find the documentation of *Morphir.SDK* [here](https://package.elm-lang.org/packages/finos/morphir-elm/latest/Morphir-SDK-List#leftJoin). The difference is that the `leftJoin` returns a `List ( a, Maybe b)` instead, so in the projection you must to return a maybe value:
+
+```elm
+simpleLeftJoin : List TypeA -> List TypeB -> List {x: Int, y: Maybe Int}
+simpleLeftJoin dataSetA dataSetB =
+     dataSetA
+        |> leftJoin dataSetB
+            ( \a b -> a.id == b.id )
+        |> List.map (\ (x, y) ->
+        { x = x.id
+        , y = Maybe.map (\t -> t.id) y
+        })
+```
+
+The transformation to Snowpark will be:
+
+```Scala
+def simpleLeftJoin
+    (dataSetA: com.snowflake.snowpark.DataFrame)
+    (dataSetB: com.snowflake.snowpark.DataFrame)
+    (implicit sfSession: com.snowflake.snowpark.Session): com.snowflake.snowpark.DataFrame = {
+    val dataSetAColumns: deparments.Joins.TypeA = new deparments.Joins.TypeAWrapper(dataSetA)
+    val dataSetBColumns: deparments.Joins.TypeB = new deparments.Joins.TypeBWrapper(dataSetB)
+    
+    dataSetA.join(
+      dataSetB,
+      (dataSetAColumns.id) === (dataSetBColumns.id),
+      "left"
+    ).select(
+      dataSetAColumns.id.alias("x"),
+      dataSetBColumns.id.alias("y")
+    )
+  }
+```
+
+In Snowpark, when employing a join operation, It is important to note that columns with `null` values may be encountered when performing a join operation. Consequently, you might observe that the `select` projection simply yields a basic column that could potentially include `null` values. For more information about Snowpark and `Joins` you can see visit [this link](https://docs.snowflake.com/en/developer-guide/snowpark/scala/working-with-dataframes#joining-dataframes).
```

Comment:
change "...you can see visit [this link]" for "... you can see [this link]" or "... you can visit[this link]"